Assets/Scripts/EnemyPatrol.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyPatrol : MonoBehaviour
+{
+
+    [SerializeField]
+    private float moveSpeed, checkRadius, attackDelay, deathDelay, knockbackPower, knockbackHeight, knockbackTime;
```

Comment:
These are great names but I'm gonna be annoying and ask you to break this up into individually declared serialized fields with ToolTip attributes to help clarify. https://docs.unity3d.com/ScriptReference/TooltipAttribute.html